app/org/apache/spark/deploy/history/SparkFSFetcher.scala

```diff
@@ -237,7 +237,7 @@ class SparkFSFetcher(fetcherConfData: FetcherConfigurationData) extends Elephant
               null
             }
           } else {
-            val logFilePath = new Path(logPath + "_1.snappy")
+            val logFilePath = getSnappyOrNot(logPath)
             if (!shouldThrottle(logFilePath)) {
               EventLoggingListener.openEventLog(logFilePath, fs)
```

Comment:
Have you verified if EventLoggingListener.openEventLog can open all type of spark logs? 

You should add a test for this.